server/utils/constants.ts

```diff
@@ -46,6 +46,16 @@ export enum SORT_DIRECTION {
   DESC = 'desc',
 }
 
+export enum DETECTORS_QUERY_PARAMS {
+  FROM = 'from',
+  SIZE = 'size',
+  SEARCH = 'search',
+  INDICES = 'indices',
+  SORT_FIELD = 'sortField',
+  SORT_DIRECTION = 'sortDirection',
+  NAME = 'name',
+}
```

Comment:
Rather than defining in multiple places: we actually can create shared constants if we add a top-level `common/` dir that's accessible by both client and server-side. Could you add this and at least add the constants file that contains this same data (+ the defaults)? See [flow framework's common/index.ts file](https://github.com/opensearch-project/dashboards-flow-framework/blob/main/common/index.ts) for reference. 

We don't need to move them all now; but we can at least stop adding to the tech debt here.

Reply:
this is not defined multiple times? We have been using string instead of enums in our package so far. And since you commented earlier about using enum for this part, so I created these name enums.

Reply:
Right, I meant adding this + the defaults in a constants file. No blocker, just a suggestion so we can use this + other defined constants across both public and server side.